sysdig/internal/client/v2/cloud_account.go

```diff
@@ -11,7 +11,8 @@ const (
 	cloudAccountsWithExternalIDPath = "%s/api/cloud/v2/accounts?includeExternalID=true&upsert=true"
 	cloudAccountPath                = "%s/api/cloud/v2/accounts/%s"
 	cloudAccountWithExternalIDPath  = "%s/api/cloud/v2/accounts/%s?includeExternalID=true"
-	trustedCloudIdentityPath        = "%s/api/cloud/v2/%s/trustedIdentity"
+	onboardingTrustedIdentityPath   = "%s/api/secure/onboarding/v2/trustedIdentity?provider=%s"
```

Comment:
The actual source value will be the same, so this won't break any backward compatibility right?

Reply:
yes, the value is the same.